<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeWindow</class>
 <widget class="QMainWindow" name="homeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>homeWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="outputStudentMsgBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示所有学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputStudentMsgBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>输入学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>按平均分升降排序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findMsgWithStudentNumBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>据学号查询学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertStudentMsgBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>插入学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyStudentMsgBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readByfileBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>从文件中读取数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="writeAllRecordToFileBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>将所有记录写入文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退出本系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;&amp;gt;&amp;gt;&amp;gt;&amp;gt; 学生信息管理系统 &amp;lt;&amp;lt;&amp;lt;&amp;lt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyrigth pjhubs.com</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
